### PR TITLE
Webaccess frame enable disable v2

### PR DESCRIPTION
**Pull request by Itay-Lifshitz**
@mcallegari 
Hi, the following widgets' enable/disable states are changed when you check/uncheck the frame enable.

VCButton
VCSlider
VCKnob
VCCueList
VCLabel
VCClock
Please review and give feedback.

Thank you.

### CODE REVIEW

File: ui/src/virtualconsole/vcframe.cpp [hunk 1]
```diff
@@ -120,9 +120,13 @@ void VCFrame::setDisableState(bool disable)
         widget->setDisableState(disable);
         if (!disable)
             widget->adjustIntensity(intensity());
+
+        emit widget->disableStateChanged(disable);
```
Comment (mcallegari) [inline ui/src/virtualconsole/vcframe.cpp:124]: setDisableState is recursive. No need to invoke widget signal from here

**Comment by Itay-Lifshitz** [comment]
@mcallegari Can you please let me know the status of this PR?
What is preventing you from reviewing/closing this PR?

**Comment by mcallegari** [comment]
@Itay-Lifshitz I'm working on some other tasks at the moment. I'll review this when I'm done

**Comment by yestalgia** [comment]
@Itay-Lifshitz there's a few quick styling changes which would be appreciated in the Codacy report.

https://app.codacy.com/gh/mcallegari/qlcplus/pullRequest?prid=13065141

Should only take a few mins to resolve. Happy to collab if you give me access to the branch.

**Comment by mcallegari** [comment]
@Itay-Lifshitz merged, thanks :+1:

**Pull request merged**